wiki/Accuracy/en.md

```diff
@@ -8,11 +8,11 @@ Accuracy is a percentile measurement of a player's ability to hit [hit objects](
 
 ## Game modes
 
-### ![](/wiki/shared/mode/osu.png) osu!standard
+### ![](/wiki/shared/mode/osu.png) osu! mode
```

Comment:
changes like this are excessive -- it's clear that `osu!` as mentioned here is a game mode, not the game in general. most of the time you want it to be `osu!standard -> osu!` kind of fix